src/program/db/db_functions.py

```
@@ -10,6 +10,7 @@
 from program.utils import root_dir
 
 import alembic
+from program.utils import root_dir
```

Ruff

Redefinition of unused `root_dir` from line 10

Remove definition: `root_dir`

(F811)